packages/backend/setupJest.ts

```diff
@@ -1,22 +1,8 @@
 import { enableFetchMocks } from 'jest-fetch-mock';
-import { LightdashAnalytics } from './src/analytics/LightdashAnalytics';
 import {lightdashConfigMock} from "./src/config/lightdashConfig.mock";
 
 enableFetchMocks();
 
 jest.mock('./src/config/lightdashConfig', () => ({
     lightdashConfig: lightdashConfigMock,
-}));
-
-jest.mock('./src/analytics/client.ts', () => ({
-    analytics: new LightdashAnalytics({
-        lightdashConfig: lightdashConfigMock,
-        writeKey: 'notrack',
-        dataPlaneUrl:
-            'notrack',
-        options: {
-            enable: false,
-        }
-    }),
-    identifyUser: jest.fn(),
-}));
+}));
```

Comment:
Should this have an empty final newline?

Reply:
We never had a preference or lint rule for it. We can adjust prettier/eslint to enforce one going forward if you want.